InteractiveAlert/InteractiveAlert.Droid/InteractiveAlerts.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Android.Support.V7.App;
+
+namespace InteractiveAlert
+{
+	public static class InteractiveAlerts
```

Comment:
Can you merge two projects "InteractiveAlert.Core" and "InteractiveAlert" in project "InteractiveAlert"?

Reply:
I don't think so, but i could be missing something...

Currently the InteractiveAlert exposes the Instance to the core project of you users.
But the actual execution will be handled by the InteractiveAlerts class of the platform projects.

If i however merge these 2 projects, the platform projects will also include the InteractiveAlerts class that throws the exception (since they depend on core) and won't overwrite them.
In that case the samples break.

I could however be missing something, in which case feel free to clarify, and I'll have another look.

Reply:
I may be wrong because I switched from Xamarin to React Native. 
You can view the sharing logic in the  [Acr.UserDialogs](https://github.com/aritchie/userdialogs). The [UserDialogs](https://github.com/aritchie/userdialogs/blob/master/src/Acr.UserDialogs/Platforms/Shared/UserDialogs.cs) class uses [partial logic](https://github.com/aritchie/userdialogs/blob/master/src/Acr.UserDialogs/Platforms/ios/UserDialogs.cs). This solution reduces the amount of code and makes the code easy to use.

Reply:
Well yes, but that is because they didn't split up their platforms over different projects.

They just have a single project in their solution that compiles differently on different platforms.
So if we want to use partial classes as well, we have to restructure the entire solution?